backend/app/adapter/sqldb/short_link.go

```diff
@@ -94,42 +94,28 @@ WHERE "%s"=$1;`,
 }
 
 // CreateShortLink inserts a new ShortLink into short_link table.
-func (s ShortLinkSQL) CreateShortLink(shortLink entity.ShortLink) error {
+func (s ShortLinkSQL) CreateShortLink(shortLinkInput entity.ShortLinkInput) error {
 	statement := fmt.Sprintf(`
-INSERT INTO "%s" ("%s","%s","%s","%s","%s","%s","%s","%s","%s","%s","%s")
-VALUES ($1, $2, $3, $4, $5, $6, $7, $8, $9, $10, $11);`,
+INSERT INTO "%s" ("%s","%s","%s","%s")
+VALUES ($1, $2, $3, $4);`,
 		table.ShortLink.TableName,
 		table.ShortLink.ColumnAlias,
 		table.ShortLink.ColumnLongLink,
 		table.ShortLink.ColumnExpireAt,
 		table.ShortLink.ColumnCreatedAt,
-		table.ShortLink.ColumnUpdatedAt,
-		table.ShortLink.ColumnOpenGraphTitle,
-		table.ShortLink.ColumnOpenGraphDescription,
-		table.ShortLink.ColumnOpenGraphImageURL,
-		table.ShortLink.ColumnTwitterTitle,
-		table.ShortLink.ColumnTwitterDescription,
-		table.ShortLink.ColumnTwitterImageURL,
 	)
 	_, err := s.db.Exec(
 		statement,
-		shortLink.Alias,
-		shortLink.LongLink,
-		shortLink.ExpireAt,
-		shortLink.CreatedAt,
-		shortLink.UpdatedAt,
-		shortLink.OpenGraphTags.Title,
-		shortLink.OpenGraphTags.Description,
-		shortLink.OpenGraphTags.ImageURL,
-		shortLink.TwitterTags.Title,
-		shortLink.TwitterTags.Description,
-		shortLink.TwitterTags.ImageURL,
+		shortLinkInput.CustomAlias,
```

Comment:
```suggestion
		shortLinkInput.GetCustomAlias(""),
```
I think this can't be nil.

Reply:
👍 I think you're right. The column is non-nullable in both `short_link` and `user_short_link` tables.